include/exadg/utilities/input_parameters.h

```diff
@@ -0,0 +1,53 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2021 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef INCLUDE_EXADG_UTILITIES_INPUT_PARAMETERS_H_
+#define INCLUDE_EXADG_UTILITIES_INPUT_PARAMETERS_H_
+
+// deal.II
+#include <deal.II/base/parameter_handler.h>
+
+namespace ExaDG
+{
+struct InputParameters
```

Comment:
The struct ``GridData`` has already been prepared for reading grid files with a string for the filename. So I think there is no need to intoduce another struct right now.

Reply:
I forgot about the the struct in `GridData`.  I will change the design accordingly.

Reply:
addressed in ce5731610269cac685066d2a69b823c059586cd3.